src/locales/ko/battle-scene.ts

```diff
@@ -0,0 +1,5 @@
+import { SimpleTranslationEntries } from "#app/interfaces/locales";
+
+export const battleScene: SimpleTranslationEntries = {
+  "moneyOwned": "{{formattedMoney}}원"

```

Comment:
```suggestion
  "moneyOwned": "₽{{formattedMoney}}"
```
Korean currency is '원'
But we are using ₽ at other locale texts

https://github.com/pagefaultgames/pokerogue/blob/main/src/locales/ko/modifier-select-ui-handler.ts#L12

https://github.com/pagefaultgames/pokerogue/blob/main/src/locales/ko/battle.ts#L16